go/cmd/main.go

```diff
@@ -0,0 +1,139 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Canhassi12/scylla-cloud-getting-started/internal/controllers"
+	"github.com/Canhassi12/scylla-cloud-getting-started/internal/database"
+	"github.com/joho/godotenv"
+	"github.com/scylladb/gocqlx/v2"
+)
+
+func main() {
+	godotenv.Load(".env")
+	
+	session, err := database.Connect(); 
+	if err != nil {
+		println("error database connection =>", err.Error())
+		return
+	}
+
+	migratePath := os.Getenv("MIGRATE_PATH")
+
+	if err := executeMigrateFile(session, migratePath); err != nil {
+		println(err.Error())
+		os.Exit(1)
+	}
+
+	reader := bufio.NewReader(os.Stdin)
+
+	songController := controllers.NewSongController(session)
+	stressController := controllers.NewStressController(session)
+
+	for {
+		fmt.Print("\nAvailable commands:\n!add - add a new song\n!list - list all registered songs\n!delete - delete a specific song\n!stress - stress testing with mocked data\n!q - Quit the console\n\n")
+		
+		input, err := reader.ReadString('\n')
+		if err != nil {
+			fmt.Println("Error to read the input:", err)
+			break
+		}
+
+		command := strings.TrimSpace(input)
+
+		switch command {
+		case "!add":
+			fmt.Println("Insert song title: ")
+			titleInput, err := reader.ReadString('\n')
+			if err != nil {
+				fmt.Println("Error to read the input:", err.Error())
```

Comment:
I would say its probably safe to ignore error in this case with `_`. Since you're reading from stdin, and its inteded for user interactive shell, this will never give an error. It would look more clean ignoring the errors here.

Reply:
Maybe a good option is to create a function that prints "error start again" and restart the inputs

Reply:
@lalizita that would be nice, but @racerxdl is right. I wasn't actually aware of this, but I tested and since it is reading from stdin it doesn't show any error. It is weird, but I guess that in this case omitting the error is acceptable.